src/main/java/org/sagebionetworks/bridge/services/AppService.java

```diff
@@ -339,7 +340,12 @@ public App createApp(App app) {
         
         Study study = Study.create();
         study.setAppId(app.getIdentifier());
-        study.setIdentifier(app.getIdentifier() + "-study");
+        // For legacy reasons, these two default apps use non-standard initial study IDs. 
+        if (BOOTSTRAP_STUDY_IDS.contains(app.getIdentifier())) {
+            study.setIdentifier(app.getIdentifier());
+        } else {
+            study.setIdentifier(app.getIdentifier() + "-study");    
+        }
```

Comment:
I don't think this is actually necessary. For api, the Integ Tests will immediately replace the default study with with study1, and the shared study doesn't actually have any participants, so study doesn't matter.

Reply:
Hmm okay, my integration tests seemed to fail but I will test this out again.